projects/components/src/skeleton/skeleton.component.ts

```diff
@@ -0,0 +1,65 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+interface ContainerClass {
+  skeleton: boolean;
+  'skeleton-rectangle': boolean;
+  'skeleton-square': boolean;
+  'skeleton-circle': boolean;
+  'skeleton-rectangle-text': boolean;
+  'skeleton-table-row': boolean;
+  'skeleton-list-item': boolean;
+  'skeleton-repeating': boolean;
+}
+
+export const enum SkeletonType {
+  Rectangle = 'rectangle',
+  RectangleText = 'rectangle-text',
+  Square = 'square',
+  Circle = 'circle',
+  TableRow = 'table-row',
+  ListItem = 'list-item'
+}
+
+@Component({
+  selector: 'ht-skeleton',
+  template: `
+    <ng-container *ngFor="let k of [].constructor(this.repeat)">
```

Comment:
Can we move this array construction into TS for readability (ideally just do something like `this.repeat && new Array(this.repeat)` Also, be careful with undefined values here.